matlab_octave/bci_workshop_tools/libsvm_binaries/README

```diff
@@ -0,0 +1,245 @@
+-----------------------------------------
```

Comment:
Copied the [COPYRIGHT](https://github.com/cjlin1/libsvm/blob/master/COPYRIGHT) file to the `libsvm_binaries` folder